frontend/lang/italiano.json

```
@@ -0,0 +1,311 @@
+    "t_panel_status_short": "Stato",
+    "t_panel_docs": "Documentazione",
+    "t_panel_docs_description": "Documentazione: Cerca la documentazione di Julia e ricevi documentazione in tempo reale su tutto quello che digiti.",
+    "t_panel_status_progress": "Stato {{progress}}",
+    "t_panel_status_progress_inner": "({{done}}/{{total}})",
+    "t_panel_close": "Chiudi pannello",
+    "t_panel_popout": "Separa pannello",
+    "t_stop_notebook_session": "Ferma sessione",
+    "t_shut_down_notebook": "Spegni notebook",
+    "t_start_notebook_in_background": "Avvia notebook in background",
+    "t_remove_from_recent_notebooks": "Rimuovi dai notebook recenti. Questo non elimina il file del notebook.",
+    "t_loading_something": "Caricamento {{text}}...",
+    "t_loading_something_notebook_from_clipboard": "notebook dagli appunti (clipboard)",
+    "t_loading_something_new_notebook": "nuovo notebook",
+    "t_open_a_notebook_action": "Apri un notebook",
+    "t_enter_path_or_url": "Inserisci path o URL...",
+    "t_open_file_action": "Apri file",
+    "t_open_action": "Apri",
+    "t_open_from_url_action": "Apri da URL",
+    "t_close_notebook_session": "Chiudere la sessione del notebook?",
+    "t_shut_down_notebook_process": "Spegnere il processo del notebook?",
+    "t_loading_ellipses": "Caricamento...",
+    "t_notebooks": "Notebook",
+    "t_cell_input_placeholder": "Inserisci codice della cella...",
+    "t_add_cell": "Aggiungi cella ({{key}})",
+    "t_drag_to_move_cell": "Trascina per spostare la cella",
+    "t_show_hide_code": "Mostra/nascondi codice",
+    "t_cell_disabled_in_file_tooltip": "Questa cella è contrassegnata come disabilitata nel file. Clicca per saperne di più!",
+    "t_cell_indirectly_disabled_in_file_tooltip": "Questa cella è indirettamente contrassegnata come disabilitata nel file. Clicca per saperne di più!",
+    "t_cell_disabled_in_file_explanation": "Questa cella è attualmente memorizzata nel file del notebook come <em>commento</em>, invece che come <em>codice</em>.<br />In questo modo, il codice della cella non verrà eseguito quando il notebook viene usato come script al di fuori di Pluto.<br />Usa il menu contestuale per riabilitarla.",
+    "t_cell_indirectly_disabled_in_file_explanation": "Questa cella è attualmente memorizzata nel file del notebook come <em>commento</em>, invece che come <em>codice</em>.<br />In questo modo, il codice della cella non verrà eseguito quando il notebook viene usato come script al di fuori di Pluto.<br />Una cella a monte sta <b>indirettamente</b> <em>disabilitando nel file</em> questa; abilita {{upstreamLink}} per influenzare questa cella.",
+    "t_language_picker_description": "Cambia la lingua dell'interfaccia",
+    "t_refresh_to_see_language_change_confirm": "Lingua cambiata! Vuoi aggiornare la pagina per vedere le modifiche?",
+
+    "t_anonymous_function_abbr": "Una (mini-)funzione che è definita senza la parola chiave 'function', ma usando -> o 'do'.",
```

Honestly I was confused that "o" was a variable name part of the anonymous function, I'd suggest using "oppure" instead, to make it less ambiguous:
```suggestion
    "t_anonymous_function_abbr": "Una (mini-)funzione che è definita senza la parola chiave 'function', ma usando -> oppure 'do'.",
```